src/utils/hsdis/capstone/hsdis-capstone.c

```diff
@@ -52,6 +52,7 @@
 #include <inttypes.h>
 #include <string.h>
 
```

Comment:
Need a comment about this `undef`

Reply:
Added a comment.

Reply:
Please also update the copyright year for this file.

Reply:
Updated